### PR TITLE
feat: add speakers page

### PR DESCRIPTION
**Pull request by tobySolutions**
I was able to:
- Create and properly setup the speakers page
- Implement the speakers' card
- Add a few speakers that were on the Figma given to me

Please help review @santoshyadavdev, @geromegrignon. Thank you!!

Screenshot 2024-03-22 at 15 33 29

### CODE REVIEW

**Comment by santoshyadavdev** [comment]
Can you remove my profile  I am not speaking at the event, that's just a sample

**Comment by tobySolutions** [comment]
> Can you remove my profile I am not speaking at the event, that's just a sample

Yeah, sure thing! Let me do that.

**Comment by tobySolutions** [comment]
> Can you remove my profile I am not speaking at the event, that's just a sample

Done!

File: src/components/SpeakerCard.astro [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+    }
+
+    .title {
+        color: white;
+        -webkit-text-stroke: 1px #21005D;
+    }
+
+
+
+/* @media (min-width: var(--big_screen)) {
```
Comment (geromegrignon) [inline src/components/SpeakerCard.astro]: to be removed if you do not need it

**Pull request merged**